Stylesheets/components/input.css

```diff
@@ -0,0 +1,61 @@
+@import url("/stylesheets/main.css");
+
+.input-holder {
+  width: fit-content;
+  margin: 1rem;
+  padding: 0.5rem;
+  background-color: var(--bg-black);
+  color: white;
```

Comment:
use the variable of white color you created here for consistency

Reply:
oh yes right i did changes